plugins/parsers/avro/README.md

```diff
@@ -58,6 +58,16 @@ The message is supposed to be encoded as follows:
   #      }
   #'''
 
+  ## login and password in format "Login:Password" encoded to base64 string
+  ## that will be added as a basic auth header to the schema registry requests
+  ## Should be filled only when schema registry uses basic authentication 
+  ## For example "Aladdin:open sesame" equals to "QWxhZGRpbjpvcGVuIHNlc2FtZQ==" in base64
+  # avro_schema_registry_authorization_base64 = "QWxhZGRpbjpvcGVuIHNlc2FtZQ=="
+
+  ## Path to the schema registry certificate. Should be specified only when it is 
+  ## required for connection to the schema registry. 
+  # avro_schema_registry_cacert_path = "/etc/telegraf/ca_cert.crt"
+
```

Comment:
Please move those options up to right below `avro_schema_registry` to have all options prefixes `avro_schema_registry` close to each other.

Furthermore, I strongly vote for splitting `avro_schema_registry_authorization_base64` into `avro_schema_registry_username` and `avro_schema_registry_password` and doing the encoding in the plugin. Otherwise this is cumbersome to use...